src/PhpSpreadsheet/Writer/Html.php

```diff
@@ -596,6 +596,12 @@ private function extendRowsForChartsAndImages(\PhpOffice\PhpSpreadsheet\Workshee
         }
         $html = '';
         ++$colMax;
+
+        //Return $html if $rowMax didn't change
+        if ($row !== $rowMax) {
+            return $html;
```

Comment:
`$html` didn't change between initialization (line 597) and first usage. Maybe return hard-coded empty string instead?